### DISCUSSION TITLE
Suppressing output ... integrity and index procs

### DISCUSSION THREAD

**Discussion by hipybri** [olahallengren/sql-server-maintenance-solution#563] [Questions & Answers]
We are executing the maintenance solution in a enterprise scheduler built on java (via some dynamic sqlcmd executions).  The scheduler examines the output for specific error messages to tell if the job executes correctly or not.  However, from time to time there is just too much output from the integrity or index procedures.  

Does anyone know how to output only errors to stdout for these procedures?  We log all of the commands to commandlog (logtotable = 'y'), so we capture the executions and don't need them in stdout/stderr.

**Comment by olahallengren** [comment]
There is currently no option to only output errors.

One thing to keep in mind is that is not possible to log all errors to the table.
This is the case with the backup and restore commands that produces two errors.
Please see details in the Frequently Asked Questions, https://ola.hallengren.com/frequently-asked-questions.html.
It is also the case with the consistency check commands.
